### ISSUE TITLE
Set numbers shaders to show nothing outside game

### ISSUE THREAD

**Issue by looooph** [alex-ong/NESTrisSharpener#3]
As described in topic

Especially when used in tournament play, it looks "untidy", as it tries to make sense of the tetris background, showing random numbers(see picture below)

![image](https://user-images.githubusercontent.com/40400000/92315739-a3efaa00-efea-11ea-8af9-b43b8c3df6ae.png)

**Comment by alex-ong** [comment]
Use nestrisautolayout to fix this

**Comment by alex-ong** [comment]
Color correct
Nestrissharpener
Nestrissharpener numbers (probably need 3-5)
Nestrisautolayout

Edit (this order is wrong, read below)

**Comment by looooph** [comment]
Ah, I wasn't even aware of the layout one. But can't use it due to it not yet being Linux compatible.

**Comment by alex-ong** [comment]
Is the tournament over yet? So i know whether i should prioritize anything; also your hydrantdude same rng patch is a little out of date; feel free to visit CTM for latest one

**Comment by looooph** [comment]
Yes, it was the german championship, happening two days ago. But as I'm one of the very few linux users in the community, there is no hurry at all. But I'll gladly assist in testing once there is a linux version of the autolayout.

**Comment by looooph** [comment]
Did some testing, loooph was diligent. If I got this right the autolayout is designed to work with unmodded stencil ready scenes only, so I set everything up accordingly. Still I found two things not displaying correctly:
On Press start screen, the sharpened level shows.
On Music select screen, the sharpened current score(not top) shows the upper 5 digits.

Added pictures to clarify
![image](https://user-images.githubusercontent.com/40400000/92422517-fca37c00-f17d-11ea-9ac2-9085d4f09936.png)

**Comment by alex-ong** [comment]
I got the filter order wrong; sorry. But once you get the order correct, you should be able to verify that inside the field area everything is correct, with the "wrong numbers" showing up outside. After I add "blank" as a valid number in the algo it should all work.

note order:
Make Stencil ready scene
Apply **scene** filters
* Nestrissharpener (blocks/field)
* NestrisAutolayout
* sharpen numbers (3x) (all these areas are outside the field; autolayout puts all other content inside the field)

Final Scene: crops from stencil ready scene that has filters

You should get garbage/randoms but they'll all be outside field area.

Then once we patch it to recognize black == show black, it should work.
